yaml-tests/src/main/java/com/apple/foundationdb/relational/yamltests/command/QueryExecutor.java

```diff
@@ -258,6 +260,10 @@ private Object executeStatementWithForcedContinuations(final @Nonnull Statement
                         Assertions.assertFalse(hasNext);
                     }
                 }
+                count += 1; // PMD failure for ++
+                if (count > MAX_CONTINUATIONS_ALLOWED) {
```

Comment:
This seems fine for now. In theory, we could try and assert on things like whether we've advanced the continuation or something more obscure, but we probably don't want to have too many tests that exceed 100 continuations anyway